projects/igniteui-angular-i18n/src/i18n/NB/paginator-resources.ts

```diff
@@ -0,0 +1,16 @@
+import { IPaginatorResourceStrings } from 'igniteui-angular';
+
+// exported below as re-cast to create declaration type with expanded properties
+const PaginatorResourceStringsNB_: ExpandRequire<IPaginatorResourceStrings> = {
+    igx_paginator_label: 'Elementer per side',
+    igx_paginator_pager_text: 'av',
+    igx_paginator_first_page_button_text: 'Gå til første side',
+    igx_paginator_previous_page_button_text: 'Forrige side',
+    igx_paginator_last_page_button_text: 'Gå til siste side',
+    igx_paginator_next_page_button_text: 'Neste side',
```

Comment:
Maybe a template* leftover, however it will be better to remove the excess coma at the end